### ISSUE TITLE
Maintain this repository

### ISSUE THREAD

**Issue by nhducit** [shichuan/javascript-patterns#154]
Hello everybody,

This repository seems to be inactive for a quite long time. It has 20 open issues and 39 Pull Requests. That's why I create a new github organization to fork and maintain this repository. 
Here is it: https://github.com/javascript-patterns/javascript-patterns

If you want to maintain this repository, please let know by leaving comment on this issue.

Thanks.

**Comment by glebinsky** [comment]
I can review some of the open PRs and create PRs your fork if you'd like.

**Comment by roblarsen** [comment]
FWIW, the fairest way (in terms of credit) would be to merge the PRs locally, pulling from the original submitter's remote branch and then push the results to the new remote. Creating new PRs loses the original author.

**Comment by roblarsen** [comment]
That's if you can't get ahold of @shichuan to see if you can getvownership of the repo transfered over, of course.  Taking over the project directly would be best.

**Comment by glebinsky** [comment]
Yeah, I meant merging the old PRs into new branches; certainly wasn't going to c/p the original changes. I'm fine with just PRing to your fork since the last change to his repo was 3 years ago.

**Comment by nhducit** [comment]
@glebinsky @roblarsen 
Would you mind if I invite you join the javascript-pattarns organization?
It would be awesome if you can help to review pull requests and resolve PRs.

**Comment by glebinsky** [comment]
I wouldn't mind but can't promise to always be available.

**Comment by landonsanders** [comment]
I'd like to maintain it.

**Comment by landonsanders** [comment]
I'd like to maintain it. I am always available.

**Comment by sg1asgard** [comment]
It would be a pity to loose this repository. I found it very useful and would be great if you guys can take car of it.

**Comment by roblarsen** [comment]
@nhducit can you add me as a collaborator on the cloned repo?

**Comment by nhducit** [comment]
Added @roblarsen @glebinsky @landonsanders as admin of the cloned repo

**Comment by jamesjara** [comment]
I want to help too @nhducit

**Comment by vasanthk** [comment]
@nhducit I would like to help maintain this as well.

**Comment by KirankumarAmbati** [comment]
The repo: https://github.com/javascript-patterns/javascript-patterns 
is 140 commits behind this repo. Is that repo is also not maintained ?? Can I help you ? I would like to be one of the maintainer.

@shichuan can I assist you in maintaining this repo ?? I would be happy to do so. 😃

**Comment by nhducit** [comment]
@KirankumarAmbati invited you
